test/e2e/playwright/swap/support/extension-page.ts

```diff
@@ -0,0 +1,59 @@
+import path from 'path';
```

Comment:
why duplicating this file? does the other not works for your tests?

Reply:
Yes you are right. Old code here. Forgot to delete